src/queue/Queue.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * The queue package contains our Queue, QueueTest, QueueNode and QueueTest 
+ * classes.
+ * 
+ * @author Scott McDonald and Jon Cronin
+ */
+package queue;
+
+/**
+ * Queue implements the basic Queue ADT, with the functions enque() and deque().
+ * Users of this queue class can set the maximum capacity or accept the default
+ * of 10.
+ * @author Scott McDonald and Jon Cronin
+ */
+public class Queue {
+    private QueueNode head;
+    private QueueNode tail;
+    private int capacity;
+    private int size;
+    
+    // Define the return values the Queue class methods may use
```

Comment:
Basically my JUnit skills are still weak and enums in Java are actually mini classes. When i tried it as an enum type there were some functions that i needed to overload and test or jacoco wouldn't give me 100% test coverage anymore. So I abandoned the enum class type error return method for the time being. It's an enhancement I could look at adding again in the future.